server/http/restapi/asset.go

```diff
@@ -0,0 +1,27 @@
+package restapi
+
+import (
+	"net/http"
+
+	"github.com/mattermost/mattermost-plugin-apps/apps"
+	"github.com/mattermost/mattermost-plugin-apps/server/utils/httputils"
+)
+
+func (a *restapi) handleGetStaticAsset(w http.ResponseWriter, req *http.Request, actingUserID string) {
+	query := req.URL.Query()
+
+	assetName := query.Get("name")
+	appID := query.Get("app_id")
+
+	// TODO verify that request is from the correct app
```

Comment:
Why do we need to check for that? If an app dev uses static assets to as an icon in a modal, there won't be an app connected to that request, correct?

Reply:
Gentle bump on that question

Reply:
We just using user tokes for the authorization so "malicious" app can get a static asset of a different app. It's not a problem for now but might become later.

Reply:
Let me ask the question in another way: Do we consider the content of static assets to be confidential information?

Reply:
I think so. There might be some data files in the static folder, not just an icon.

Reply:
I am confused... `/static` is just public images, nothing "private" about it AFAIC.